README.md

```diff
@@ -175,31 +175,31 @@ Install-Module InvokeBuild -Scope CurrentUser
 Install-Module platyPS -Scope CurrentUser
 ```
 
-### 4. Delete `NuGet.Config`
+### 4. Adjust `Nuget.Config` if necessary
 
-Our NuGet configuration points to a private feed necessary for secure builds,
-and it must be committed to the repo as it is.
-The easiest way to build without access to that private feed is to delete the file:
+Our NuGet configuration uses a secure feed with whitelisted third party dependency packages. If your contribution requires any changes to the included NuGet packages, you must disable this secure feed.
```

Comment:
```suggestion
Our NuGet configuration uses a secure feed with allow-listed third party dependency packages. If your contribution requires any changes to the included NuGet packages, you must disable this secure feed.
```
I know it's silly but our pipeline will flag us for "whitelist" and "blacklist" it has to be "allow-list" and "deny-list" 😆

Reply:
Not at all! Force of habit I probably need to change anyways.